### PR TITLE
Czech translation of Getting Started project

### PR DESCRIPTION
**Pull request by dvorapa**
I asked on the IRC channel about number of characters per line. I found out english version keeps maximum of 118 characters per line, so I followed this limit, but I could rework it if it wouldn't be okay. I also added a screenshot, but I haven't got a large screen, so feel free to update it with yours (please  make it using build with the czech translation for v 1.4 - #11347 - there were some significant changes in order to correspond with the Getting Started project translation)

### CODE REVIEW

**Comment by dvorapa** [comment]
I asked on the IRC channel abput number of characters per line. I found out english version keeps maximum of 118 characters per line, so I followed this limit, but I could rework it if it wouldn't be okay. I also added a screenshot, but I have not so large screen, so feel free to update it with yours (please  make it using build with the czech translation for v 1.4 - )

**Comment by abose** [comment]
Thanks @dvorapa For this translation. 
@martinstarman could you take a look at this?

File: samples/cs/Getting Started/index.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            správné množství pomoci, kdy budete chtít.
+        </p>
+        
+        <!--
+            CO JE BRACKETS?
+        -->
+        <p>
+            <em>Brackets je jiný druh editoru.</em>
+            Brackets obsahuje některé unikátní prvky, jako rychlou úpravu, živý náhled a další, které v jiných
+            editorech pravděpodobně nenajdete. Navíc je Brackets napsáno v JavaScriptu, HTML a CSS. To znamená,
```
Comment (martinstarman) [inline samples/cs/Getting Started/index.html]: There is little typo. Correct version is `Navíc je Brackets napsán v JavaScriptu, HTML a CSS.` Just drop the letter `o`.

**Comment by martinstarman** [comment]
Everything looks good, except one typo above.

**Comment by dvorapa** [comment]
right, corrected

**Comment by abose** [comment]
Thanks, @dvorapa . Could you merge the translation with master and submit the updated PR. There seem to be some merge conflicts.

**Comment by dvorapa** [comment]
Resolved in #11398

**Pull request closed without merging**